templates/system.json

```diff
@@ -4,31 +4,29 @@
   "operations": {
     "delete": {
       "method": "DELETE",
-      "url": "{{ properties.datatype }}/{{ entity.id }}"
+      "url": "{{ properties.datatype }}/{{ properties.id }}"
```

Comment:
We need to switch this to entity, that's what the share pipe expects.

Reply:
implemented.